microprofile/messaging/src/main/java/module-info.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2018, 2020 Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Microprofile metrics implementation.
+ */
+module io.helidon.microprofile.messaging {
+    requires java.logging;
+
+    requires static cdi.api;
+    requires static javax.inject;
+    requires static java.activation;
+    requires javax.interceptor.api;
+    requires io.helidon.config;
+    requires org.reactivestreams;
+    requires microprofile.reactive.messaging.api;
```

Comment:
All APIs that should be visible for the user should be `transitive` - that will make them imported to the using module as well.